model/Dataset/Properties/anonymizationMethodUsed.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# anonymizationMethodUsed
+
+## Summary
+
+The field describes the methods used to anonymyze the dataset (of fields in the dataset).
+
+## Description
+
+The field describes the anonymization methods used. If multiple anonymization methods are used, each anonymization method is specified in a new line. 
```

Comment:
Same comment as [here](https://github.com/spdx/spdx-3-model/pull/163#discussion_r1156985033).
That is, `anonymizationMethodUsed` has arbitrary cardinality when used in the [Dataset class](https://github.com/spdx/spdx-3-model/blob/dataset-profile/model/Dataset/Classes/Dataset.md), so we should not discuss multiple values in this property.